app/src/main/java/com/auth0/android/lock/app/DemoActivity.kt

```diff
@@ -0,0 +1,260 @@
+/*
+ * DemoActivity.java
+ *
+ * Copyright (c) 2016 Auth0 (http://auth0.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.auth0.android.lock.app
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.CheckBox
+import android.widget.LinearLayout
+import android.widget.RadioGroup
+import androidx.appcompat.app.AppCompatActivity
+import com.auth0.android.Auth0
+import com.auth0.android.authentication.AuthenticationException
+import com.auth0.android.callback.Callback
+import com.auth0.android.lock.*
+import com.auth0.android.lock.utils.LockException
+import com.auth0.android.provider.WebAuthProvider.login
+import com.auth0.android.provider.WebAuthProvider.logout
+import com.auth0.android.result.Credentials
+import com.google.android.material.snackbar.Snackbar
+import java.util.*
```

Comment:
Why not import just the utils you need, like in the Java file?

Reply:
`java.util.*` was, in fact, an unused import. I've removed it. I've also tried to simplify the `lock.*` one but the IDE preferred that one after importing a few individual classes and re-organizing the imports.